lib/sidekiq/cron/job.rb

```diff
@@ -202,13 +204,20 @@ def self.load_from_array! array
       end
 
       # get all cron jobs
-      def self.all
+      def self.all(namespace = Sidekiq::Cron.configuration.default_namespace)
```

Comment:
if this were `self.all(namespace = nil)` and would mean `ignore namespaces` it would seem more logical to me

Reply:
I understand you point, but I then don't see how users will use the `default_namespace`.